proxy/src/http/org.rs

```diff
@@ -145,6 +192,47 @@ impl ToDocumentedType for registry::Org {
     }
 }
 
+impl Serialize for registry::Project {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut state = serializer.serialize_struct("Project", 3)?;
+        state.serialize_field("name", &self.name.to_string())?;
+        state.serialize_field("orgId", &self.org_id.to_string())?;
+        state.serialize_field("maybeProjectId", &self.maybe_project_id)?;
+
+        state.end()
+    }
+}
+
+impl ToDocumentedType for registry::Project {
+    fn document() -> document::DocumentedType {
+        let mut properties = std::collections::HashMap::with_capacity(3);
+        properties.insert(
+            "name".into(),
+            document::string()
+                .description("Name of the project")
+                .example("monadic"),
+        );
+        properties.insert(
+            "orgId".into(),
+            document::string()
+                .description("The id of the org")
+                .example("monadic"),
+        );
+        properties.insert(
+            "maybeProjectId".into(),
+            document::string()
+                .description("The id project attested in coco")
+                .example("monadic")
+                .nullable(true),
```

Comment:
The examples for the project name and project id should not be the same as the org id for clarification.